src/data/meetings.js

```diff
@@ -60,6 +64,7 @@ export default {
     remoteShare: null,
     state: MeetingState.JOINED,
     showRoster: true,
+    showSettings: true,
```

Comment:
is this changing the UI somewhere?

Reply:
Yes, this change is visible in the storybook. In this case, in the corresponding story in WebexMeeting, the settings control will be initially in the ACTIVE state and will be red.